mathesar_ui/src/components/cell-fabric/data-types/components/SteppedInputCell.svelte

```diff
@@ -168,15 +169,17 @@
   {#if isEditMode}
     <slot {handleInputBlur} {handleInputKeydown} />
   {:else}
-    <div
-      class="content"
-      class:nowrap={!isActive && !isIndependentOfSheet}
-      class:truncate={isActive && multiLineTruncate && !isIndependentOfSheet}
-    >
-      <slot name="content" {value} {formatValue} {matchParts}>
-        <CellValue value={formattedValue} {matchParts} />
-      </slot>
-    </div>
+    <Truncate passthrough={!truncate}>
+      <span
+        class="content"
+        class:nowrap={!isActive && !isIndependentOfSheet}
+        class:truncate={isActive && multiLineTruncate && !isIndependentOfSheet}
+      >
+        <slot name="content" {value} {formatValue} {matchParts}>
+          <CellValue value={formattedValue} {matchParts} />
+        </slot>
+      </span>
+    </Truncate>
```

Comment:
@Ary0405 Notice in line 176 that the component only truncates values if the conditions specified there are met. You're enclosing the entire block within the Truncate component and enabling it by default, which will break usage. 

1. When the component is used independently of the sheet, it should not get truncated. eg., in Data explorer, upon selecting a cell, you can notice the content in the Cell tab in the right pane, which is truncated. It shouldn't be truncated there. The prop tracking this is `isIndependentOfSheet `.
2. `multiLineTruncate` prop is used to check if the truncation should take multiple lines into account. That logic needs to be handled using the `lines` prop from Truncate component.

Please get rid of the existing truncate related classes i.e. `nowrap` and `truncate`, and handle all logic using the Truncate component. Here's a rudimentary suggestion:
```suggestion
    <div class="content">
      <Truncate
        lines={isActive && multiLineTruncate ? 2 : 1}
        passthrough={isIndependentOfSheet}
      >
        <slot name="content" {value} {formatValue} {matchParts}>
          <CellValue value={formattedValue} {matchParts} />
        </slot>
      </Truncate>
    </div>
```